### PR TITLE
Kits/Hooks: Fix - Not saving globals inside 'Site Settings' (Fix PT#1…

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

**Comment by iNewLegend** [comment]
Create common class for shared logic between the hooks.

File: core/kits/assets/js/hooks/data/globals/base-globals-update.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+	getRepeaterName() {
+		elementorModules.ForceMethodImplementation();
+	}
+
+	applyModel( model, id, value ) {
+		elementorModules.ForceMethodImplementation();
+	}
+
+	apply( args, result ) {
```
Comment (matipojo) [inline core/kits/assets/js/hooks/data/globals/base-globals-update.js:18]: Please provide unit tests.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/kits/assets/js/hooks/data/globals/base-globals-update.js (hunk 1 above)
Comment (iNewLegend) [reply core/kits/assets/js/hooks/data/globals/base-globals-update.js:18]: Handled.

File: assets/dev/js/editor/commands/close.js [hunk 2]
```diff
@@ -14,8 +14,8 @@ export class Close extends CommandBase {
 			return jQuery.Deferred().resolve();
 		}
 
-		// TODO: Move to an hook.
-		if ( ! mode && ( document.editor.isChanged || document.isDraft() ) ) {
+		// TODO: Move to an hook ( UI Hook not under tests ).
+		if ( ! elementorCommonConfig.isTesting && ! mode && ( document.editor.isChanged || document.isDraft() ) ) {
```
Comment (matipojo) [inline assets/dev/js/editor/commands/close.js:18]: Remove `elementorCommonConfig.isTesting`.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/commands/close.js (hunk 2 above)
Comment (iNewLegend) [reply assets/dev/js/editor/commands/close.js:18]: Simple removing of 'elementorCommonConfig.isTesting' will trigger test error, the solution is move it to an hook, i will do  and create new UI hook for handling this.

Comment (iNewLegend) [reply assets/dev/js/editor/commands/close.js]: handled.

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**